.github/workflows/titles.yml

```diff
@@ -2,7 +2,10 @@
 name: Validate Titles
 
 on:
+  # zizmor: ignore[dangerous-triggers] Limits to maintained main branch, and avoids running PR code
```

Comment:
The comment below claims that base/main is used, but I don't see that in the code.

Reply:
See https://github.com/OSGeo/grass/pull/6296#discussion_r2320692119